src/components/Accordion/index.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { ReactNode, useState } from 'react';
+import TestBlock from '../TestBlock';
+import {
+  AccordionHeader,
+  AccordionHeaderContent,
+  AccordionItem,
+  AccordionRoot,
+  AddIcon,
+  CancelIcon,
+  CheckCircleIcon,
+  RemoveIcon,
+  StyledAccordionContent,
+  StyledAccordionTrigger
+} from './style';
+
+interface Props {
+  children: ReactNode;
+  'data-state'?: string;
+  failed?: string[];
+  value?: string;
+}
+
+const TestFileView: React.FC = () => {
+  const [failed, setFailed] = useState(['item-1', 'item-2']);
+
+  return (
+    <AccordionRoot type="multiple" value={failed} onValueChange={(data) => setFailed(data)}>
+      <AccordionItem value="item-1">
+        <AccordionTrigger failed={failed} value="item-1">
+          DuckDuckGo
+        </AccordionTrigger>
+        <AccordionContent>
+          <TestBlock>Test title with lengthy text will be wrapped in second line</TestBlock>
+        </AccordionContent>
+      </AccordionItem>
+      <AccordionItem value="item-2">
+        <AccordionTrigger failed={failed} value="item-2">
+          Is it accessible?
+        </AccordionTrigger>
+        <AccordionContent>
+          <TestBlock>Find Nightwatch JS in duckDuckgo</TestBlock>
+        </AccordionContent>
+      </AccordionItem>
+      <AccordionItem value="item-3">
+        <AccordionTrigger failed={failed} value="item-3">
+          Is it accessible?
+        </AccordionTrigger>
+        <AccordionContent>Yes. It adheres to the WAI-ARIA design pattern.</AccordionContent>
+      </AccordionItem>
+      <AccordionItem value="item-4">
+        <AccordionTrigger failed={failed} value="item-4">
+          Is it accessible?
+        </AccordionTrigger>
+        <AccordionContent>Yes. It adheres to the WAI-ARIA design pattern.</AccordionContent>
+      </AccordionItem>
+      <AccordionItem value="item-5">
+        <AccordionTrigger failed={failed} value="item-5">
+          Is it accessible?
+        </AccordionTrigger>
+        <AccordionContent>Yes. It adheres to the WAI-ARIA design pattern.</AccordionContent>
+      </AccordionItem>
+    </AccordionRoot>
+  );
+};
+
+const AccordionTrigger = React.forwardRef<HTMLButtonElement, Props>(
```

Comment:
Please create separate files for different components.

Reply:
Since it's a part of Accordion, IMO, it should stay here. Also, it's a reusable component. 
PS: I am using Headless UI (i.e. Radix UI)